app/views/declaration/declarationHolder/declaration_holder_edit_content.scala.html

```diff
@@ -71,7 +71,7 @@
             label = messages("declaration.declarationHolder.authorisationCode"),
             labelClass = Some("govuk-label--m"),
             hintParagraphs = declHolderHelper.hintForAuthorisationCode,
-            emptySelectValue = messages("declaration.declarationHolder.authorisationCode.empty"),
+            emptySelectValue = Some(messages("declaration.declarationHolder.authorisationCode.empty")),
```

Comment:
Why are we keeping this particular 'empty' value (where the others are being removed)?